tests/lava/tutorials/test_tutorials.py

```diff
@@ -216,7 +216,7 @@ def test_end_to_end_01_mnist(self):
             "tutorial01_mnist_digit_classification.ipynb", e2e_tutorial=True
         )
 
-    @unittest.skipIf(system_name != "linux", "Tests work on linux")
+    @unittest.skip('Running the notebook takes too long')
```

Comment:
We might need some special tests for tutorials then, otherwise broken tutorials will end up in releases. @mgkwill any ideas?

Reply:
In NxSDK, long tests are only executed in dedicated runs during the night. Maybe that would be an option?